src/table/__tests__/inline-editor.test.tsx

```diff
@@ -203,24 +204,32 @@ describe('InlineEditor', () => {
   });
 
   // useless test but need the coverage
-  it('should render spinner while submitting', () => {
+  it('should render spinner while submitting', async () => {
     thereBeErrors = false;
-    const changeEvent = new Event('change', { bubbles: true });
     const { wrapper } = renderComponent(<TestComponent />);
+    let resolveSubmit = () => {};
+    handleSubmitEdit.mockImplementationOnce(() => {
+      return new Promise<void>(resolve => {
+        resolveSubmit = resolve;
+      });
+    });
 
     const input = wrapper.find('input')!.getElement();
     fireEvent.click(input);
-    fireEvent(input, changeEvent);
+    act(() => {
+      fireEvent.change(input, { bubbles: true, target: { value: 'new value' } });
+    });
```

Comment:
Did you check if `act` and `bubbles: true` really needed? I think they are already included inside `fireEvent` by default